plugin/src/main/java/io/snyk/eclipse/plugin/properties/PreferencesPage.java

```diff
@@ -71,13 +56,17 @@ protected void createFieldEditors() {
     addField(new StringFieldEditor(Preferences.ADDITIONAL_PARAMETERS, "Additional Parameters:", getFieldEditorParent()));
     addField(new StringFieldEditor(Preferences.ADDITIONAL_ENVIRONMENT, "Additional Environment:", getFieldEditorParent()));
 
+    addField(space());
+    addField(new BooleanFieldEditor(Preferences.MANAGE_BINARIES_AUTOMATICALLY, "Update and install Snyk binaries automatically", getFieldEditorParent()));
+    addField(new FileFieldEditor(Preferences.LS_BINARY_KEY, "Snyk LS Path:", getFieldEditorParent()));
```

Comment:
This should be `Language Server Path` or `Path to Language Server`

Reply:
Could also be `Snyk Language Server` to use the same naming scheme as `Snyk CLI Path`. But LS is a short acronym that we shouldn't use here.

Reply:
done